packages/scipy/patches/fix-blas.patch

```diff
@@ -0,0 +1,89 @@
+commit f03d85c00ae993c3c87a5a7b2ea0c1215f57ee04
+Author: Roman Yurchak <rth.yurchak@pm.me>
+Date:   Mon Oct 8 10:53:11 2018 +0200
+
+    Partial fixes for BLAS/LAPACK
+
+diff --git a/scipy/linalg/setup.py b/scipy/linalg/setup.py
+index 2c9b9ba22..e8c7336d5 100755
+--- a/scipy/linalg/setup.py
++++ b/scipy/linalg/setup.py
+@@ -14,7 +14,13 @@ def configuration(parent_package='', top_path=None):
+ 
+     config = Configuration('linalg', parent_package, top_path)
+ 
+-    lapack_opt = get_info('lapack_opt')
++    lapack_opt = {  # libraries will be auto-generated by pywasmcross
++                  'libraries': [],
++                  'include_dirs': [],
++                  'library_dirs': ['../../../../CLAPACK/CLAPACK-WA/'],
++                  'language': 'f77',
++                  'define_macros': [('NO_ATLAS_INFO', 1),
++                                    ('HAVE_CBLAS', None)]}
```

Comment:
Here we hard code the values that would normally be determined by distutils. The latter would also usually have `'libraries': ['blas', 'lapack']` for all files that need them which get transformed into `-lblas -llapack`, while here we detect the `-L../[..]/CLAPACK/CLAPACK-WA/` argument and statically link BLAS/LAPACK as needed.